examples/preprocessors__typescript-webpack/README.md

```diff
@@ -19,26 +19,23 @@ The E2E tests should run in Cypress right away. There are few other commands con
 
 ## Notes
 
-Cypress includes its own TypeScript definitions starting with version 1.1.3. For older versions of Cypress you will need to install them separately using `npm i -D @types/cypress`.
-
-To get Mocha TypeScript definitions for `describe`, `it` and other BDD functions install the mocha typings. To get definitions for `expect` install chai typings.
+Cypress includes its own TypeScript definitions starting with version 1.1.3.
 
+For older versions of Cypress you will need to install them separately.
 ```
-npm install --save-dev @types/mocha @types/chai
+npm i -D @types/cypress @types/mocha @types/chai
 ```
 
 Because `cypress` is installed in the root folder of this repo, [tsconfig.json](tsconfig.json) specifies where to find its types:
 
 ```
 {
-  "include": [
-    "../../node_modules/cypress/index.d.ts",
-    "cypress/*/*.ts"
-  ]
+  "types": ["cypress"],
```

Comment:
interesting - this seems to be equivalent to `/// reference types="cypress"` trick

Reply:
Yes. The added benefit over `"includes"` is to isolate the types being used in the Cypress directory. I've seen some people have issues with type conflicts if they happen to use namespace types (globals) in their application code like `chai` or `jquery`. Just having "cypress" will use the "cypress" types as the root and the Cypress type definitions will pull in dependencies.